tools/devops/automation/templates/variables.yml

```diff
@@ -35,3 +35,7 @@ variables:
   value: 'https://devdiv.visualstudio.com/_settings/agentpools?poolId=367&view=agents'
 - name: CIBuild
   value: true
+- name: azureStorage
+  value: 'bosstoragemirror'
+- name: azureContainer
+  value: 'wrench'
```

Comment:
should this be a location other than wrench?

Reply:
I was told the container was not going to be changed, so I left it as is.